src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1069,22 +1069,28 @@ export function getNextEvents(currentStatus, details) {
  * @returns {alertOutput} dynamically-generated title, description, and styling properties
  */
 export function getAlertContent(alert) {
-  const { type } = alert;
+  const { type, details, date, dueDate } = alert;
+  // TODO: confirm new date/dueDate usages
+  const formattedDate = moment(date, 'YYYY-MM-DD').format('MMMM DD, YYYY');
+  const formattedDueDate = moment(dueDate, 'YYYY-MM-DD').format('MMMM DD, YYYY');
```

Comment:
Are these used appropriately in the updated content?

Reply:
`date` and `dueDate` actually come from the `alert.details` object, but other than that, it looks good. That's what the issue you referenced in the original comment is talking about.